src/components/MapImageSlider.vue

```diff
@@ -44,14 +552,13 @@
 </template>
 <script>
 import BeerSlider from "beerslider";
-import cartogram from "../assets/cartogram/cartogram_init_tooltip.svg";
+/* import cartogram from "../assets/cartogram/cartogram_init_tooltip.svg"; */
 import GeorgiaInsetMap from './georgiaInsetMap';
 import atlantaSliderText from "../assets/mapboxSlider/atlantaSliderText";
 export default {
     'name': 'MapImageSlider',
     components:{
-        cartogram, 
-        GeorgiaInsetMap
+          GeorgiaInsetMap
```

Comment:
I am not a Vue expert, so I am struggling with this concept. Do we not need to reference the `cartoUSA` div here?

Reply:
We do not need to because the svg is inline, but you are correct in your thinking. When I initially put the map in this component I referenced the external file you produced from R and used `cartogram` here to import and then place using html. Since we determined the inline svg to be better suited, I deleted that piece and commented out the other lines that were involved.

Reply:
I am going to merge for now!

Reply:
As I expected, it works so I just don't have the right understanding about components yet :)